src/lib/units/offset.js

```diff
@@ -161,11 +161,10 @@ export function setOffsetToLocal (keepLocalTime) {
 }
 
 export function setOffsetToParsedOffset () {
-    if (this._tzm) {
+    if (this._tzm === 0 || this._tzm) {
```

Comment:
Here's the thing. I dont know what this if is supposed to check for. We either write the cases explicitly or documented it above.